core/input/generisdata_emi.prn

```diff
@@ -35,8 +35,9 @@ pecoal.seliqfos.coalftrec      26.1
 pecoal.seliqfos.coalftcrec      7.8      18.3
 pecoal.sesofos.coaltr          26.1             0.04415
 pecoal.segafos.coalgas         26.1             0.04415
-pebiolc.seel.bioigccc         -20.0      20.0   
-pebiolc.seel.bioigcc                            
+pebiolc.seel.bioigccc         -20.0      20.0
+pebiolc.seel.bioigcc
+pebiolc.segabio.biogasc        -9.5       9.5   0.12614
```

Comment:
With [this bugfix](https://github.com/remindmodel/remind/pull/885) we removed the N2O emission factors from generisdata_emi.prn for pebiolc technologies completely.
```suggestion
pebiolc.segabio.biogasc        -9.5       9.5
```